# request
string name
---
# response
float64[] poses
float64[] jacobians
